commerce_coordinator/apps/titan/pipeline.py

```diff
@@ -130,34 +133,38 @@ class CreateDraftPayment(PipelineStep):
     def run_filter(
         self,
         order_uuid,
-        response_code,
+        payment_intent_id,
+        client_secret,
         payment_method_name,
         provider_response_body,
+        edx_lms_user_id,
     ):  # pylint: disable=arguments-differ
         """
         Execute a filter with the signature specified.
         Args:
-            Args:
             order_uuid(str): Order UUID related to this order.
-            response_code(str): Payment attempt response code (payment intent id) provided by stripe.
             payment_method_name(str): The name of the payment method used for this payment. See enums for valid values.
             provider_response_body(str): The response JSON dump from a request to the payment provider.
-
+            payment_intent_id(str): A Stripe Payment Intent ID (used to update payment intents)
+            client_secret(str): A Stripe client secret string used by the UI to load the Stripe payment form.
+            edx_lms_user_id(str): edC LMS User ID
         """
 
         api_client = TitanAPIClient()
         try:
             payment = api_client.create_payment(
                 order_uuid=order_uuid,
-                response_code=response_code,
+                response_code=payment_intent_id,
```

Comment:
this response_code should also be renamed to refernceNumber, As per [API spec](https://github.com/edx/commerce-coordinator-private/blob/main/docs/api-spec/theseus-api-spec.md#input-10), responseCode does not exist.